src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+export type Tab = {
+  id: string
+  title: string
+  content: string
+};
+
+type TabsProps = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  const selectedTabContent = () => tabs
```

Comment:
why do you need function here?
```suggestion
  const selectedTabContent = tabs
```

Reply:
I deleted function.